assets/charts/control-plane/kube-apiserver/values.yaml

```diff
@@ -9,7 +9,7 @@ apiserver:
   aggregationCaCert:
   aggregationClientCert:
   aggregationClientKey:
-  image: k8s.gcr.io/kube-apiserver:v1.18.8
```

Comment:
This should close issue #866

Reply:
Yep this is already added in the PR description.

Reply:
I don't like putting it in the commit message because it pollutes the issue workspace for every force push.